examples/dependency/README.md

```diff
@@ -10,7 +10,7 @@ kubectl apply -f examples/dependency/definition.yaml
 Start OAM runtime:
 
 ```shell script
-go run ./cmd/oam-runtime/main.go
+go run cmd/oam-kubernetes-runtime/main.go
```

Comment:
Using [the installation step](https://github.com/crossplane/oam-kubernetes-runtime#install-oam-runtime)